src/sql/load_vocabulary.sql

```diff
@@ -130,25 +130,25 @@ TRUNCATE concept_class CASCADE;
 TRUNCATE domain CASCADE;
 
 -- Temporary fix to include CPT4 concepts.
-\COPY CONCEPT FROM PROGRAM 'unzip -p /data/vocabulary/vocab.zip CONCEPT_CPT4.csv | awk ''BEGIN { FS="\t"; OFS="\t" } ; NR==1 || $2 {print;next}; {print $1, "CPT4 - " $7, $3, $4, $5, $6, $7, $8, $9, $10}''' WITH DELIMITER E'\t' CSV HEADER QUOTE E'\b' ;
+\COPY CONCEPT FROM '/data/vocabulary/CONCEPT_CPT4.csv | awk ''BEGIN { FS="\t"; OFS="\t" } ; NR==1 || $2 {print;next}; {print $1, "CPT4 - " $7, $3, $4, $5, $6, $7, $8, $9, $10}''' WITH DELIMITER E'\t' CSV HEADER QUOTE E'\b' ;
```

Comment:
This still needs to be PROGRAM
```suggestion
\COPY CONCEPT FROM PROGRAM 'awk ''BEGIN { FS="\t"; OFS="\t" } ; NR==1 || $2 {print;next}; {print $1, "CPT4 - " $7, $3, $4, $5, $6, $7, $8, $9, $10}'' CONCEPT_CPT4.csv' WITH DELIMITER E'\t' CSV HEADER QUOTE E'\b' ;
```

Reply:
You are absolutely right about this point. But you still need to write `/data/vocabulary/CONCEPT_CPT4.csv` instead of `CONCEPT_CPT4.csv`!